src/util/ApiClient.tsx

```diff
@@ -0,0 +1,38 @@
+import entityNames from '~/enums/entityNames';
+
+enum RequestMethod {
+    POST= 'POST',
+    PUT= 'PUT',
+    DELETE= 'DELETE',
+}
+
+const API_URL = process.env.API_URL || 'http://localhost:3040';
```

Comment:
add / to the end

Reply:
Added it instead to row 36, thought that would be more natural